src/Php80/bootstrap.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Symfony\Polyfill\Php80 as p;
+
+if (PHP_VERSION_ID < 80000) {
+    if (!function_exists('fdiv')) {
```

Comment:
As the polyfill is not compatible with PHP 5 (while the main package is), I think we should add a version check here. Not registering the polyfilled function is better than registering it but throwing an exception when using it. Code checking for `function_exists('fdiv')` would break if we register a non-working polyfill.